python/python/lance/log.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright The Lance Authors
+
+import logging
+import os
+from typing import Optional
+
+ENV_NAME_PYLANCE_LOGGING_LEVEL = "PYLANCE_LOGGING_LEVEL"
+
+logger = logging.getLogger("pylance")
+logger.setLevel(os.environ.get(ENV_NAME_PYLANCE_LOGGING_LEVEL, "INFO"))
+
+
+def set_logger(
```

Comment:
Here, we open the ability for users to configure the pylance's logger to align with themself.